csrc/options.h

```diff
@@ -108,6 +108,7 @@ enum class EnableOption {
   WaitDebugger, // Used for debugging multi-GPU. The rank given in the argument
                 // will wait for `gdb attach` at the start.
   WarnRegisterSpill, //! Enable warnings of register spill
+  WarpSpecializedPersistent, //! Enable warp specialized persistent kernel
```

Comment:
Why can't we have a scheduler parameter to enable warp specialized persistent kernels? 

In `MatmulParams`, the `TilingStrategy` option allows us to select persistent kernel. The `CircularBufferingStrategy` selects between warp specialized and pipelined circular buffering.

https://github.com/NVIDIA/Fuser/blob/main/csrc/scheduler/matmul_heuristic.h#L256-L264
https://github.com/NVIDIA/Fuser/blob/main/csrc/scheduler/matmul_heuristic.h#L278-L283

Reply:
Parameterized testing would be easier if warp specialization was a scheduler choice.

Reply:
Thanks for the suggestion! We can and will do that after the heuristic development is done.
For parameterized testing, I extended the test to run with `EnableOption::WarpSpecializedNormalization` set and unset.

Reply:
Actually, these knobs will be added to scheduler params in step-2.